tasks/ssh_private_key.yaml

```diff
@@ -0,0 +1,18 @@
+- name: ensure ssh private key is set
+  loop: |
+    {%- set private_keys = [] -%}
+    {%- for user_name, user_value in system.users.items() -%}
+      {%- if user_value.private_key is defined -%}
```

Comment:
Must be defined and not none.

Reply:
With this nothing is changed if not private key is set.